book/content/material/material_advanced.md

```diff
@@ -0,0 +1,273 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.3
+kernelspec:
+  display_name: festim-workshop
+  language: python
+  name: python3
+---
+
+# Advanced Material Features #
+
++++
+
+This tutorial introduces more advanced ways of handling materials in FESTIM. A key addition in FESTIM 2.0 is the ability to perform multi-species simulations, which often requires assigning distinct material properties to each species (see __[documentation](https://festim.readthedocs.io/en/v2.0-alpha.3/api/index.html#festim.material.Material)__ for more information). In some cases, users may also need to define custom material properties—for example, specifying a turbulence-dependent viscosity or user-defined diffusivity.
+
+Objectives:
+* Assign different material properties to different species
+* Solve a problem with custom material properties (e.g., diffusivity)
+
++++
+
+## Assigning different material properties in a multi-species simulation ##
+
+Consider the following 1D example that simulates the diffusion of protium, deuterium and tritium species:
```

Comment:
I think that if you write "Consider this" then that means you think this is important. In that case, it shouldn't be hidden. imo